core/fortran-example/tutorial/06_kokkos_parallel_for/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+  if("${CMAKE_Fortran_COMPILER_ID}" STREQUAL "PGI")
```

Comment:
`${}` not needed, plus indentation.

Reply:
@junghans ${} appears to be needed. Otherwise, the 06 example does not compile.